hass-game-db-updater/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ghcr.io/andrew-codes/playnite-web-game-db-updater:3-dev
```

Comment:
I attempted to confine addons to their own directory, but unfortunately, that's not an available option. You need to define a basic repository.json in the root of the repository, and that’s the extent of the control you have. HASS will check out the repository, scan the first-level directory tree for directories containing config.json, and then read all the configurations from there. One of the configurations specifies which image to use.

Another approach is to create a separate repository specifically for handling HASS addons, which seems cleaner to me.

Reply:
Thanks for the info, which is exactly what I was looking for. Given this is the case, we could make a separate repo for HASS add-ons, but one downside is that there is a dependency between the two repos; i.e., release happens first in this repo and then second in the HASS add-on one. This makes automation a little harder, though possible. It is akin to coordinating a single logical change across many repos.

For this reason, I'd prefer to keep them in the same mono-repo. Also, as noted elsewhere in this PR, having a centralized version management mechanism could help here, too, whereas it would be less helpful across repos.

With all this said, here is what I'd prefer to see:

- [ ] directory structure remains as it is in the PR. This is a hard requirement from HASS.
- [ ] Each of these will have a `project.json` and `package.json`. This will enable repo-wide management of tasks, such as coordinating build, packaging, and publish orderings.
    - [ ] There are no build targets for either, but there are package and publish targets that will be needed.
        - package to build image and publish to publish image to ghcr.io
     - [ ] `package.json` only has dev deps on Nx related modules.

I can work on the central version management as a separate item. I would plan to merge this PR before the version management one. For this reason, I'm not too worried about the version changing from 3-dev. I think it is fine as it is until the version management work is completed; that will naturally address the versioning here.

Also, I'm more than happy to help on any of the above. I have tried to hold off because I didn't want to step on toes in case you wanted to do this work. I'll focus efforts on the versioning as this is clearly needed sooner than later at this point. Thanks again! I'm looking forward to making Playnite-Web awesome with you and others in the community!